locales/es.lua

```diff
@@ -9,14 +9,20 @@ local Translations = {
     },
     info = {
         ["refuel_vehicle"] = "Rellenar vehículo",
-        ["gas_station"] = "Estación de gasolina",
+        ["take_nozzle"] = "Tomar la boquilla",
+        ["return_nozzle"] = "Devolver la boquilla",
+        ["gas_station"] = "Estación de servicio",
```

Comment:
Put this back as

  ["gas_station"] = "Estación de gasolina",

Reply:
Why tho? In all spanish countries we do not call it "Estacion de gasolina", it's an "Estación de servicio".

Reply:
https://es.wikipedia.org/wiki/Estaci%C3%B3n_de_servicio

Reply:
I speak Spanish and and the correct term is Estacion de gasolina, if you want this PR merged this change needs to take effect else, feel free to close the PR.

Reply:
No, it's not. I live in Costa Rica and have lived in Argentina, Venezuela, México, and the US even. That's not the term used. You already closed it yourself.

Reply:
That's fine, we don't want wrong PRs that are incorrect.